### PR TITLE
Fix #2052: Math.nextUp(0.0) == Double.MinPositiveValue

### LINKED ISSUE #2052: java.lang.Math.{nextUp|upl} bug using Doble.{MinValue|MIN_VALUE}.
On JVM (at least in JDK8) `nextUp(0.0)` and `nextUp(-0.0)` return `4.9E-324` (the smalest larger representable double) instead of the [documented](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#nextUp-double-) `Double.MinValue`. The same happens with the the `0.0f` and `-0.0f`.

### PR DESCRIPTION
**Pull request by soc**
Scala.MinValue, it should have been Scala.MinPositiveValue

### CODE REVIEW

File: javalanglib/src/main/scala/java/lang/Math.scala [hunk 1]
```diff
@@ -102,11 +102,11 @@ object Math {
     if (a != a || a == PositiveInfinity)
       a
     else if (a == NegativeInfinity)
-      -MaxValue
+      MinValue
```
Comment (nicolasstucki) [inline javalanglib/src/main/scala/java/lang/Math.scala]: Why this change? The documentation says that it should be `-MaxValue`. Have tested it on the JVM?

**Comment by sjrd** [comment]
Could you make the title of the commit message fit on one line? Suggestion:

    Fix #2052: Math.nextUp(0.0) == Double.MinPositiveValue.

    Details here.

File: javalanglib/src/main/scala/java/lang/Math.scala (hunk 1 above)
Comment (nicolasstucki) [inline javalanglib/src/main/scala/java/lang/Math.scala]: Oh, I see, they are the same. I was just following the spec word by word.

**Comment by soc** [comment]
Updated.

**Comment by sjrd** [comment]
LGTM. Jenkins, test this please

**Comment by sjrd** [comment]
retest this please

File: test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/lang/MathTest.scala [hunk 2]
```diff
@@ -176,19 +176,19 @@ object MathTest extends JasmineTest {
       expect(Math.nextAfter(0.0, -0.0)).toEqual(-0.0)
       expect(Math.nextAfter(-0.0, 0.0)).toEqual(0.0)
       expect(Math.nextAfter(-0.0, -0.0)).toEqual(-0.0)
-      expect(Math.nextAfter(Double.MinValue, Double.NegativeInfinity)).toEqual(Double.NegativeInfinity)
-      expect(Math.nextAfter(-Double.MinValue, Double.PositiveInfinity)).toEqual(Double.PositiveInfinity)
+      expect(Math.nextAfter(Double.MinPositiveValue, Double.NegativeInfinity)).toEqual(Double.NegativeInfinity)
+      expect(Math.nextAfter(-Double.MinPositiveValue, Double.PositiveInfinity)).toEqual(Double.PositiveInfinity)
```
Comment (sjrd) [inline test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/lang/MathTest.scala]: These two lines should not have changed:
```scala
scala> Math.nextAfter(Double.MinValue, Double.NegativeInfinity)
res2: Double = -Infinity

scala> Math.nextAfter(-Double.MinValue, Double.PositiveInfinity)
res3: Double = Infinity
```

**Comment by nicolasstucki** [comment]
@soc, can you also rebase on top of master.

**Comment by soc** [comment]
Addressed @sjrd's comment and rebased as @nicolasstucki suggested.

**Comment by sjrd** [comment]
LGTM. Jenkins, retest this please
